doc/features/index.rst

```diff
@@ -0,0 +1,32 @@
+.. _features-overview:
+
+Features
+========
+
+Arbor is fully featured, ready for real world modelling, and the user community is busy collaborating with us to build complex models. Best practices, tips, tricks and howtos will be documented here as the community gains experience.
+
+Features include:
+
+* :ref:`in_gui`: a fully featured GUI for single cell construction.
+* :ref:`Support for a range of fileformats <format-overview>`, including :ref:`formatnmodl` and NeuroML by way of `NMLCC <https://github.com/arbor-sim/nmlcc>`_, a fully featured NeuroML converter for Arbor targets.
+* `Nascent SONATA format support <https://github.com/arbor-sim/arbor-sonata>`_. The Allen mechanism database :ref:`is already included in Arbor <mechanisms_builtins>`, so currently, with some manual work executing Allen models is already possible, see :ref:`tutorialsinglecellallen`.
+* :ref:`Voltage Processes <formatnmodl_voltageproc>` in the form of a mechanism kind. Useful for e.g. implementing voltage clamps.
+* :ref:`Stochastic Differential Equations <tutorial_calcium_stpd_curve>`: both point and density mechanisms may now use white noise as part of the state updates, turning the ODEs effectively into SDEs.
+* :ref:`Mutable connection table <interconnectivity-mut>`: the connection table can be redefined between successive simulation function calls.
+* :ref:`Edit morphologies <morph-edit>` using Arbor API, useful for e.g. merging segment trees.
+* `BluePyOpt integration <https://github.com/BlueBrain/BluePyOpt/releases/tag/1.14.0>`_. Parameter optimization using BluePyOpt can be done through an Arbor backend and exported to an Arbor cable cell, see :ref:`this tutorial <tutorialsinglecellbluepyopt>`.
+* :ref:`Wide range of cable cell probes<pycablecell-probesample-api>`, :ref:`plus LIF cell probes <pycablecell-probesample-lif>`.
+* :ref:`Inhomogeneous parameters <cablecell-scaled-mechs>`: enables :ref:`painting of scaled density mechanisms <labels-iexpr>`, e.g. with the distance from the root. (nmodl: ``iexpr``.)
+* :ref:`LFPyKit integration <tutorial_lfpykit>`: extract extracellular signals from an Arbor cable cell.
+* :ref:`Faster NMODL programming guide <formatnmodl-faster>`: helps users write NMODL with performance and fewer bugs in mind.
```

Comment:
That's not really a feature ... maybe we make a list of helpful things a later further on?

Reply:
It's indeed not a pure list of neuroscientific features, but I thought it worth mentioning those features that will come in handy when modelling. I'll move this one down at the bottom, where I put others falling in this category.